pyworkflowtests/tests/test_object.py

```diff
@@ -30,11 +30,10 @@
 
 import pyworkflow.object as pwobj
 import pyworkflow.tests as pwtests
-import pyworkflow.utils as pwutils
+from objects import Complex, MockSetOfImages, MockImage, MockObject, \
```

Comment:
Please, follow PEP8, don't use \ to break lines

Reply:
Thanks @delarosatrevin , I never tried the "python -m unittest discover" command.

Now it works too. I've addressed the other issues.